README.md

```diff
@@ -75,6 +75,36 @@ Options:
 
 The `-cert` option may be omitted if the `-key` option contains both a PEM-encoded certificate and key.
 
+## Running in Docker
+
+We have included a Dockerfile so you can easily build and run keywhiz-fs with all of its dependencies. To build a kewhizfs Docker image run the following command:
+
+```
+docker build --rm -t keywhizfs .
+```
+
+After building, you can run the newly built image by running:
+
+```
+docker run --device /dev/fuse:/dev/fuse --cap-add=IPC_LOCK --cap-add=SYS_ADMIN keywhizfs -debug=true -ca=/go/src/github.com/square/keywhiz-fs/fixtures/cacert.crt -key=/go/src/github.com/square/keywhiz-fs/fixtures/client.pem https://localhost:443 /secrets/kwfs
+```
+
+Note that we have to pass `--device /dev/fuse:/dev/fuse` to mount the fuse device into the container, and give `IPC_LOCK` and `SYS_ADMIN` capabilities to the container, so it can set `cap_ipc_lock` on the keywhiz-fs binary, and so it can mount fuse-fs filesystems, respectively.
+
+This build mounts the fuseFS at `/secrets/kwfs/`.
+
+### Caveats
+
+Currently keywhiz-fs is not a [12 factor](http://12factor.net/) application, and it does not send unbuffered logs to stdout. It currently expects there to be a syslog server being ran locally.
```

Comment:
Well, it should at least support the option of sending output to syslog. It is a small change and allows a ton of other use-cases for which we didn't think keywhiz-fs would be used for.

I think it is the right thing, and will send a PR soon.